packages/atomic/src/components/atomic-search-interface/atomic-search-interface.tsx

```diff
@@ -235,7 +242,34 @@ export class AtomicSearchInterface {
       return;
     }
 
-    this.engine.executeFirstSearch();
+    const analyticsAction = this.checkForStandaloneSearchBoxData();
+    this.engine.executeFirstSearch(analyticsAction);
+  }
+
+  private checkForStandaloneSearchBoxData() {
+    const standaloneSearchBoxData =
+      new SafeStorage().getParsedJSON<StandaloneSearchBoxData | null>(
+        StorageItems.STANDALONE_SEARCH_BOX_DATA,
+        null
+      );
+
+    if (!standaloneSearchBoxData) {
+      return;
+    }
+
+    const {updateQuery} = loadQueryActions(this.engine!);
+    const {logSearchFromLink, logOmniboxFromLink} = loadSearchAnalyticsActions(
+      this.engine!
+    );
+
+    const {value, analytics} = standaloneSearchBoxData;
+    const {cause, metadata} = analytics;
+
+    this.engine!.dispatch(updateQuery({q: value}));
+
+    return cause === 'searchFromLink'
+      ? logSearchFromLink()
+      : logOmniboxFromLink(metadata!);
```

Comment:
This code can be removed if using `executeFirstSearchAfterStandaloneSearchBoxRedirect`. @jpmarceau, perhaps we can update the guide too?

Reply:
Oh right forgot about that method! :D